modules/lvgl/input/lvgl_pointer_input.c

```diff
@@ -46,15 +54,6 @@ static void lvgl_pointer_process_event(const struct device *dev, struct input_ev
 		return;
 	}
 
-	/* adjust coordinates */
```

Comment:
Is there any benefit I fail to see for moving this up into the switch? I'd prefer the compactness of the previous state.